src/ports/web/api/tests.rs

```diff
@@ -91,21 +91,34 @@ fn create_place_with_tag_duplicates() {
 }
 
 #[test]
-fn create_place_with_sharp_tag() {
+fn create_place_with_sharp_tag_and_custom_link() {
     let (client, db) = setup();
-    let json = r##"{"title":"foo","description":"blablabla","lat":0.0,"lng":0.0,"categories":["x"],"license":"CC0-1.0","tags":["foo","#bar"]}"##;
+    let json = r##"{"title":"foo","description":"blablabla","lat":0.0,"lng":0.0,"categories":["x"],"license":"CC0-1.0","tags":["foo","#bar"],"links":[{"url":"example.com","title":"Auto-completed URL"}]}"##;
     let response = client
         .post("/entries")
         .header(ContentType::JSON)
         .body(json)
         .dispatch();
     assert_eq!(response.status(), Status::Ok);
     test_json(&response);
-    let tags = db.exclusive().unwrap().all_places().unwrap()[0]
-        .0
-        .tags
-        .clone();
-    assert_eq!(tags, vec!["bar", "foo"]);
+    let (place, _) = db
+        .shared()
+        .unwrap()
+        .all_places()
+        .unwrap()
+        .into_iter()
+        .next()
+        .unwrap();
+    assert_eq!(&place.tags[..], &["bar", "foo"]);
+    // The "https://www." prefix should be added implicitly by auto-completion!
+    assert_eq!(
+        &place.links.unwrap().custom[..],
```

Comment:
dear @uklotzde 

the field name of custom links for posting data is `links`, however, the name of the same field on the get request is `custom`. this inconsistency leads to confusion. could I kindly ask you to fix that at your convenience? and when that so notify me so I would be able to integrate front end properly?

thank you so much for being such a great developer